Makefile

```diff
@@ -83,11 +83,13 @@ globals:
 .PHONY: dev
 dev: globals ## Set Environment to DEV
 	$(eval export AWS_ACCOUNT=dev)
+	$(eval export PERSISTENT_ENVIRONMENT=false)
 	$(eval export ENABLE_BILLING_APP ?= false)
 	$(eval export ENABLE_DESTROY=true)
 	$(eval export ENABLE_AUTODELETE=true)
 	$(eval export SYSTEM_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipeline.digital)
 	$(eval export APPS_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipelineapps.digital)
+	$(eval export ALERT_EMAIL_ADDRESS=govpaas-alerting-dev@digital.cabinet-office.gov.uk)
```

Comment:
Is this likely to get super-spammy? Can we pull out email addresses from an environment variable or sub-address this with our environment name so we can write email filters?

Reply:
It will have more spam than Monty Python at the Hollywood Bowl, but this is an email address made for spam. We don't expect developers to subscribe to this email group unless they are doing so for debugging purposes.

Reply:
/me unsubscribes from email group

Reply:
@samcrang Protip: you can remain subscribed to the group, but set it to not email you.

Reply:
I have it configure with a daily digest in case I ever need something to read.